packages/lesswrong/lib/collections/users/custom_fields.ts

```diff
@@ -130,6 +130,20 @@ const partiallyReadSequenceItem = new SimpleSchema({
   },
 });
 
+export const CAREER_STAGES = [
+  {value: 'highSchool', label: "In high school"},
+  {value: 'associateDegree', label: "Pursuing an associate's degree"},
+  {value: 'undergradDegree', label: "Pursuing an undergraduate degree"},
+  {value: 'professionalDegree', label: "Pursuing a professional degree"},
+  {value: 'graduateDegree', label: "Pursuing a graduate degree (e.g. Master's)"},
+  {value: 'doctoralDegree', label: "Pursuing a doctoral degree (e.g. PhD)"},
+  {value: 'otherDegree', label: "Pursuing other degree/diploma"},
+  {value: 'earlyCareer', label: "Working (0-5 years experience)"},
+  {value: 'midCareer', label: "Working (6-15 years of experience)"},
+  {value: 'lateCareer', label: "Working (15+ years of experience)"},
+  {value: 'retired', label: "Retired"},
+]
```

Comment:
This list is the same as the EAG application, except that I excluded the "Seeking work / Not currently working" option. From the EAG survey, some people found that option confusing (since the two parts mean different things). Also for the EA Forum, it seemed like the right thing to do is provide a separate "open to job offers" option to cover "seeking work". If the user is "not currently working", then they should just leave this field blank.

Reply:
I went ahead and added "Seeking work" as an option, since we're not planning on adding another field for that. I'm not really convinced that "Not currently working" is worth making the list longer.